js/ElementRendererProvider.js

```diff
@@ -410,6 +410,36 @@ export default class ElementRendererProvider {
         Utils.appendAttributesFromObject(divCarouselWrapper, config.accessibility.web);
       }
 
+      if (config.style) {
+        const style = Utils.styleToCss(config.style);
+        const splitedStyle = Utils.extractFromStyles(style, 'background-color');
+        // Set all defined style attributes on the button itself
+        // Set extractedStyle (in this case background-color) specifically on the button itself
+        arrowRight.setAttribute('style', splitedStyle.extractedStyle);
+        arrowLeft.setAttribute('style', splitedStyle.extractedStyle);
+        // Change color to border:
+        const borderStyle = Utils.styleToBorder(config.style);
+        if (borderStyle !== '') {
+          arrowRight.setAttribute('style', `${splitedStyle.extractedStyle} ${borderStyle}`);
+          arrowLeft.setAttribute('style', `${splitedStyle.extractedStyle} ${borderStyle}`);
+        }
+        // Change size to button:
+        const buttonSize = Utils.styleToButton(config.style);
+        if (buttonSize !== '') {
+          arrowRight.setAttribute('style', `${splitedStyle.extractedStyle} ${buttonSize}`);
+          arrowLeft.setAttribute('style', `${splitedStyle.extractedStyle} ${buttonSize}`);
+        }
```

Comment:
@yuliyavish Hi, what if there will be both, border style and button size? Tracing these lines in my mind I see that the second `if` will rewrite style and `borderStyle` will be missed. I would also suggest to collect all styles into string first and only then do `setAttribute`, of course if we have something in that collected style

Reply:
@powerbot15  You are right, thanks. I am changing this.